portal/src/main/webapp/WEB-INF/jsp/global/header.jsp

```diff
@@ -58,9 +58,8 @@
             $(".oncoprint_help").tipTip({defaultPosition: "right", delay:"100", edgeOffset: 25});
             
             window.cbioURL =  window.location.origin + window.location.pathname.substring(0, window.location.pathname.indexOf("/",2))+'/';
-            <%String sessionServiceUrl = (GlobalProperties.getSessionServiceUrl() == null) ? "" : GlobalProperties.getSessionServiceUrl();%>
-            <%if(!sessionServiceUrl.equals("")){%>
-            vcSession.URL ='api-legacy/proxy/session-service/virtual_study';
+            <%if(GlobalProperties.getSessionServiceUrl() != null){%>
```

Comment:
@kalletlak I think this is why the screenshot is failing. Because sessionServiceUrl can be `""`